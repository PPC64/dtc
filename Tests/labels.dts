/dts-v1/;

/ {
	foo0: foo {}

	bar0: bar1: bar {
	};
	baz0: baz1: baz = "fish";
};

/ {
	foo1: foo {}
}
