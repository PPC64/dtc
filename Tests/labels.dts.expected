/dts-v1/;

/  {

	baz = "fish";
	foo {

	};
	bar {

	};
};
